src/test/utilities/collectionUtils.ts

```diff
@@ -3,7 +3,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { AsyncCollection, toCollection } from '../../shared/utilities/asyncCollection'
+
 // TODO: what is the point of this? should it live in src/shared/utilities/collectionUtils.ts ?
```

Comment:
I guess it's just a cleaner way to write out this:
```ts
(async function* () { yield* items })()
```

Which is not particularly useful for anything but testing